portal/templates/patient_profile_create.html

```diff
@@ -104,330 +104,12 @@ <h4 class="tnth-headline left-indent-top">{{ _("New Patient") }}</h4>
     {%- endif -%}
 </form>
 <a id="redirectLink" href="">&nbsp;</a>
+{%- from "profile_macros.html" import accountCreationScript -%}
```

Comment:
Seems like the pattern is to do all the imports at the top of the file.  If there's reason to do it here, fine.